app/javascript/submission_handling/polling.js

```diff
@@ -59,11 +66,22 @@ const deleteData = function (jobId) {
 
 export const renderData = (target, resultCallback, otherRule = null) => {
     pollFetch(target, otherRule).then((result) => {
+        if (checkError(result)) {
+            reportError(result.display_error, result.id);
+        }
+
         resultCallback(result);
         deleteData(target);
     }).
     catch((error) => {
-        // This would be a network error
-        reportError(error);
+        let genericError = { "new_value_formatted": "Error",
+                             "id": target };
+        resultCallback(genericError);
+        reportError('There was a network error, please try again later or call your campus library.', target);
```

Comment:
```suggestion
        reportError('There was a network error, please try again later or call your campus library.');
```

is target needed here?

Reply:
Nope! Great catch. Surprise js-lint doesn't have the ability to catch this.

Thanks!